Cargo.toml

```diff
@@ -49,7 +46,7 @@ bytemuck = "1"
 imgui = "0.9"
 log = "0.4"
 smallvec = "1"
-wgpu = "0.14"
```

Comment:
Patch versions should not be fixed for compat reasons :) This should only be done if there is a serious issue with a previous patch version :)